package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "dependencies": {
+    "@octokit/rest": "^19.0.5",
+    "body-parser": "^1.20.1",
```

Comment:
This is bundled into express now, you don't need to install it separately.

Reply:
@humphd I was not sure if you wanted to allow the creation of multiple repos for the same user. I added a unique identifier to enable the creation of multiple repos to store photos. We can check if the user already had a repo generated using my-photohub and not allow them to create a new repo. Another option I could think of is to delete the repo generated using my-phothub and replace it with a brand-new one each time. The other option is to let the user specify the repo name.

Reply:
Gone serverless with Cloudflare workers. 😛